version/version.go

```diff
@@ -15,6 +15,9 @@ var Version = "0.37.0-dev"
 // GoVersion is the version of Go this was built with
 var GoVersion = runtime.Version()
 
+// Platform is the runtime OS and architecture of this OPA binary
+var Platform = runtime.GOOS + "/" + runtime.GOARCH
```

Comment:
Does this really convey something interesting to the user? I'd imagine if you can run the binary _at all_, you know it's the binary for your platform; if it's not, you won't see this output.

Reply:
Ignore me. Just noticed I can run `linux/arm64` using docker for mac. So this does convey something :+1:

Reply:
That's exactly why I added it. I imagine it could also be helpful to maintainers when people file bug reports and paste the output of the `version` command.